api/v1alpha1/releasestrategy_types.go

```diff
@@ -52,6 +47,43 @@ type ReleaseStrategySpec struct {
 	ServiceAccount string `json:"serviceAccount,omitempty"`
 }
 
+// Pipeline holds a reference to the release Pipeline
+type Pipeline struct {
+	// BundleReference holds a reference to the Pipeline using bundle resolver
+	// +optional
+	BundleReference BundleReference `json:"bundleReference,omitempty"`
+
+	// GitReference holds a reference to the Pipeline using git resolver
+	// +optional
+	GitReference GitReference `json:"gitReference,omitempty"`
+}
+
+// BundleReference holds a reference to the Pipeline using bundle resolver
+type BundleReference struct {
+	// Name is the name of Pipeline inside the bundle
+	// +required
+	Name string `json:"name"`
+
+	// Bundle is a reference to the Tekton bundle where to find the pipeline
+	// +required
```

Comment:
should this really be required @davidmogar ? What if I am running in debug mode?

Reply:
We will use a different approach. See here: https://github.com/redhat-appstudio/operator-toolkit/pull/18/files

That way we will allow all resolver types (including `cluster` for local Pipelines).

Reply:
Change of plans. The PipelineRef type will be added to this repo in this PR (and nothing else - the rest will come later).